xls/dslx/ir_convert/extract_conversion_order.cc

```diff
@@ -661,7 +661,18 @@ static void RemoveFunctionDuplicates(std::vector<ConversionRecord>* ready) {
       bool either_is_parametric =
           function_cr.f()->IsParametric() || subject_cr.f()->IsParametric();
 
-      if (same_fns && !either_is_proc_instance_fn && !either_is_parametric) {
+      bool both_is_parametric =
+          function_cr.f()->IsParametric() && subject_cr.f()->IsParametric();
+
+      bool equal_parametric_env = false;
```

Comment:
maybe `both_are_parametric_and_equivalent = both_is_parametric && (function_cr.parametric_env() == subject_cr.parametric_env())` would be more readable?

Reply:
Sure, I'll update it in the next commit